install.ps1

```diff
@@ -840,6 +841,42 @@ foreach ($env in $configXml.config.envs.env) {
 }
 refreshenv
 
+Write-Host "[+] Checking for custom Start Layout file..."
+$customLayoutPath = Join-Path $desktopPath "CustomStartLayout.xml"
+if ([string]::IsNullOrEmpty($customLayout)) {
+    # Download custom Start Layout file from GitHub
+    $customLayoutUrl = 'https://raw.githubusercontent.com/mandiant/flare-vm/main/CustomStartLayout.xml'
+    if (-Not (Test-Path $customLayoutPath)) {
+        Write-Host "[+] Downloading configuration file..."
+        (New-Object System.Net.WebClient).DownloadFile($customLayoutUrl, $customLayoutPath)
+    }
+} else {
+    if ($customLayout.StartsWith("http")) {
+        # Download custom Start Layout file from user-provided URL
+        Write-Host "[+] Downloading custom Start Layout file..."
+        (New-Object System.Net.WebClient).DownloadFile($customLayout, $customLayoutPath)
+    } else {
+        $customLayoutPath = $customLayout
+    }
+}
+
+# Check the Custom Start Layout file exists
+if (-Not (Test-Path $customLayoutPath)) {
+    Write-Host "`t[!] Custom Start Layout file missing: " $customLayoutPath -ForegroundColor Red
+    Write-Host "`t[-] Please download CustomStartLayout.xml from $customLayoutUrl to your desktop" -ForegroundColor Yellow
+    Write-Host "`t[-] Is the file on your desktop? (Y/N): " -ForegroundColor Yellow -NoNewline
+    $response = Read-Host
+    if ($response -notin @("y","Y")) {
+        exit 1
+    }
+    if (-Not (Test-Path $customLayoutPath)) {
+        Write-Host "`t[!] Custom Start Layout file still missing: " $customLayoutPath -ForegroundColor Red
+        Write-Host "`t[!] Exiting..." -ForegroundColor Red
+        Start-Sleep 3
+        exit 1
+    }
+}
+
```

Comment:
```suggestion
if (-Not (Test-Path $customLayoutPath)) {
    Write-Host "`t[!] Custom Start Layout file missing: " $customLayoutPath -ForegroundColor Red
    Write-Host "`t[-] No apps will be pinned to the taskbar. Do you want to continue? (Y/N): " -ForegroundColor Yellow -NoNewline
    $response = Read-Host
    if ($response -notin @("y","Y")) {
        exit 1
    }
}
```
I would remove this check. We need a `config.xml` file, but we don't necessarily need the `CustomStartLayout.xml` file. I would say if it does not exists, we just continue and don't apply any layout for the taskbar.

Reply:
Completed. 👍